<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>330</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Outlook</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Check yourself!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(36, 113, 190, 255), stop:1 rgba(243, 243, 243, 255));
</string>
  </property>
  <widget class="QPushButton" name="Button_Add_Test">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>Создать тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}
</string>
   </property>
   <property name="text">
    <string>Редактировать тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Del">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 168, 168, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 54, 54, 255), stop:1 rgba(255, 225, 225, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 225, 225, 255), stop:1 rgba(255, 54, 54, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>Удалить тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Exit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(130, 0, 195);  
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.988636, stop:0 rgba(150, 45, 255, 255), stop:1 rgba(226, 196, 255, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(130, 0, 195);  
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.988636, stop:0 rgba(169, 83, 255, 255), stop:1 rgba(246, 236, 255, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(130, 0, 195);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(246, 236, 255, 255), stop:1 rgba(169, 83, 255, 255));
}
</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Start">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(202, 205, 0, 255), stop:1 rgba(255, 255, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 236, 255), stop:1 rgba(202, 205, 0, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>Начать тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Rename">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>Переименовать тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_OK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>116</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Cancel">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>80</y>
     <width>116</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}
</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>Выбрать тест</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Remove">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 168, 168, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 54, 54, 255), stop:1 rgba(255, 225, 225, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(165, 0, 0);  
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(255, 225, 225, 255), stop:1 rgba(255, 54, 54, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Results">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}
</string>
   </property>
   <property name="text">
    <string>Результаты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_License">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 7px;
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 7px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <zorder>Button_Rename</zorder>
  <zorder>Button_Add_Test</zorder>
  <zorder>Button_Edit</zorder>
  <zorder>Button_Del</zorder>
  <zorder>Button_Exit</zorder>
  <zorder>Button_Start</zorder>
  <zorder>Button_OK</zorder>
  <zorder>Button_Cancel</zorder>
  <zorder>Button_Select</zorder>
  <zorder>textBrowser</zorder>
  <zorder>Button_Remove</zorder>
  <zorder>lineEdit</zorder>
  <zorder>Button_Results</zorder>
  <zorder>Button_License</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
